### PR TITLE
FIX Center align folder preview icons in edit panel

### LINKED ISSUE #560: Folder icon is not centred on the edit panel
pasted_image_24_07_17__4_16_pm

### PR DESCRIPTION
**Pull request by robbieaverill**
Fixes #560

### CODE REVIEW

File: client/src/containers/Editor/Editor.scss [hunk 1]
```diff
@@ -70,6 +70,11 @@
   }
 }
 
+.editor__file-preview--folder {
+  margin: auto;
```
Comment (flamerohr) [inline client/src/containers/Editor/Editor.scss]: isn't `margin: auto;` the same as `margin: 0;`?

Comment (robbieaverill) [inline client/src/containers/Editor/Editor.scss]: Nope - auto will centre align

Comment (tractorcow) [inline client/src/containers/Editor/Editor.scss]: Yes that's right; This change seems fine to me.

Comment (flamerohr) [inline client/src/containers/Editor/Editor.scss]: I mean, `auto` is useless in this case, because you have `width: 100%` as well :P

Comment (flamerohr) [inline client/src/containers/Editor/Editor.scss]: tested this out, and I think it'd be safer to target `margin-left: 0; margin-right: 0;` individually
you're removing a `margin-bottom: .76925rem;` unintentionally from that element

Comment (robbieaverill) [inline client/src/containers/Editor/Editor.scss]: Yeah it does look confusing, but the margin and width are calculated as a specific pixel width.  I could use `width: inherit` instead if that'd be less confusing?

Comment (robbieaverill) [inline client/src/containers/Editor/Editor.scss]: @flamerohr OK cool, updated and pushed

Comment (flamerohr) [inline client/src/containers/Editor/Editor.scss]: yeah, I guess `inherit` would've worked too :)

**Pull request merged**